### ISSUE TITLE
What user see by default

### ISSUE THREAD

**Issue by tstepnik** [tstepnik/planner#8]
By default user  see only task for current day.
By default user see only most important tasks, after finish them, user will see less importatn tasks.(By default 3)

**Comment by rbobko93** [comment]
It depends how you want to solve this issue and implement it. If you want to do it using backend simply, user after finishing most important tasks for the day will have to refresh the list to get less important tasks if he has any or it can be done automatically on the FE side, but I think the best possible option is to do the filtering on the BE.

**Comment by tstepnik** [comment]
Adding endpoint with all user task ordered by importance is enough ?

**Comment by rbobko93** [comment]
> Adding endpoint with all user task ordered by importance is enough ?

No, it's not. Then you will return all users task just sorted, even though you said you want to only let user see tasks for today and only the ones with most importance.
